DfE.FindInformationAcademiesTrusts/Services/Export/OfstedDataExportService.cs

```diff
@@ -0,0 +1,126 @@
+using DfE.FindInformationAcademiesTrusts.Services.Trust;
+using static DfE.FindInformationAcademiesTrusts.Services.Export.ExportColumns;
+
+namespace DfE.FindInformationAcademiesTrusts.Services.Export
+{
+    public interface IOfstedDataExportService
+    {
+        Task<byte[]> Build(string uid);
+    }
+
+    public class OfstedDataExportService(IAcademyService academyService, ITrustService trustService) : ExportBuilder("Ofsted"), IOfstedDataExportService
+    {
+        private readonly List<string> headers =
+        [
+            CommonColumnNames.SchoolName,
+            CommonColumnNames.DateJoined,
+            "Current single headline grade",
+            CommonColumnNames.BeforeOrAfterJoiningHeader,
+            "Date of Current Inspection",
+            "Previous single headline grade",
+            CommonColumnNames.BeforeOrAfterJoiningHeader,
+            "Date of previous inspection",
+            "Quality of Education",
+            "Behaviour and Attitudes",
+            "Personal Development",
+            "Leadership and Management",
+            "Early Years Provision",
+            "Sixth Form Provision",
+            "Previous Quality of Education",
+            "Previous Behaviour and Attitudes",
+            "Previous Personal Development",
+            "Previous Leadership and Management",
+            "Previous Early Years Provision",
+            "Previous Sixth Form Provision",
+            "Effective Safeguarding",
+            "Category of Concern"
+        ];
+        
+        public async Task<byte[]> Build(string uid)
+        {
+            var trustSummary = await trustService.GetTrustSummaryAsync(uid);
+ 
+            if (trustSummary is null)
+            {
+                throw new DataIntegrityException($"Trust summary not found for UID {uid}");
+            }
+
+            var academiesDetails = await academyService.GetAcademiesInTrustDetailsAsync(uid);
+            var academiesOfstedRatings = await academyService.GetAcademiesInTrustOfstedAsync(uid);
+
+            return WriteTrustInformation(trustSummary)
+                .WriteHeaders(headers)
+                .WriteRows(() =>  WriteRows(academiesDetails, academiesOfstedRatings))
+                .Build();
+        }
+
+        private void WriteRows(AcademyDetailsServiceModel[] academies, AcademyOfstedServiceModel[] academiesOfstedRatings)
+        {
+            foreach (var details in academies)
+            {
+                var ofstedData = academiesOfstedRatings.SingleOrDefault(x => x.Urn == details.Urn);
+
+                GenerateOfstedRow(details, ofstedData);
+            }
+        }
+
+        private void GenerateOfstedRow(AcademyDetailsServiceModel academy, AcademyOfstedServiceModel? ofstedData)
+        {
+            var previousRating = ofstedData?.PreviousOfstedRating ?? OfstedRating.NotInspected;
+            var currentRating = ofstedData?.CurrentOfstedRating ?? OfstedRating.NotInspected;
+
+            // School Name
```

Comment:
These comments aren't needed anymore as the code is more readable with the enums